x-pack/plugins/cases/public/components/cases_context/index.tsx

```diff
@@ -30,7 +30,7 @@ export const CasesProvider: React.FC<{ value: CasesContextProps }> = ({
   value: { owner, userCanCrud, basePath = DEFAULT_BASE_PATH, features = {} },
 }) => {
   const { appId, appTitle } = useApplication();
-  const [value, setValue] = useState<CasesContextStateValue>({
+  const [value, setValue] = useState<CasesContextStateValue>(() => ({
```

Comment:
Just curious, is adding the function here just stylistic or is it better to pass a function to `useState`?

Reply:
it is a little bit better, just because the `merge({}, DEFAULT_FEATURES, features)` function will only be executed during the first render, in the previous implementation we were executing the merge every render, even though it is only used the first time.
https://reactjs.org/docs/hooks-reference.html#lazy-initial-state